internal/cli/cmd/login/login.go

```diff
@@ -177,14 +177,20 @@ func (o *loginOptions) run(ctx context.Context) error {
 		refreshToken = ""
 	}
 
-	if err = libConfig.SaveCLIConfig(
-		libConfig.CLIConfig{
+	if o.Config.APIAddress == o.ServerAddress {
+		o.Config.BearerToken = bearerToken
+		o.Config.RefreshToken = refreshToken
+		o.Config.InsecureSkipTLSVerify = o.InsecureTLS
+	} else {
+		o.Config = libConfig.CLIConfig{
```

Comment:
```suggestion
	// If re-logging into the same API server, update the following, but leave other settings alone
	if o.Config.APIAddress == o.ServerAddress {
		o.Config.BearerToken = bearerToken
		o.Config.RefreshToken = refreshToken
		o.Config.InsecureSkipTLSVerify = o.InsecureTLS
	} else { // Otherwise, completely replace the configuration
		o.Config = libConfig.CLIConfig{
```